src/content-chunks/ios-sdk-api-intro-text.mdx

```diff
@@ -0,0 +1,7 @@
+---
+name: ios sdk intro text
```

Comment:
Do we need to add this? It seems like a placeholder and might not be immediately obvious what it's use is for.

Reply:
negatory. forgot to remove those files